### PR TITLE
Set common intake card background color to light blue in northstar

### LINKED ISSUE #8724: [North Star Applicant UI] Style "My Applications" section
#### Describe the solution presented in this issue.
- Style the "My applications" section according to the mocks.
- Remove eligibility badges from the cards in the "My applications" section

#### Link to Figma mocks:
  - [mocks](https://www.figma.com/design/RmhVJCLgAKWGENJLWWAtNS/CiviForm%3A-Components%2C-Styles?node-id=2036-12554&m=dev)

#### Add screenshot of relevant mocks for this issue here: 
Image

#### Done when:
What criteria need to be met for the issue to be finished? 
- [ ] Cards in the "My Applications" section match the mocks.
- [ ] The common intake form (AKA "Benefits Finder"), when completed, shows up in "My applications", with different styles than the other cards.
- [ ] Status badges match the mocks
- [ ] Eligibility badges have been removed from the cards in the "My applications" section

# Design updates
#### My Applications - houses all complete + in progress applications at top of program index page
-  [x] Section starts with "My applications" header 
   -   This means that there will only ever be one or two sections on the homepage: just "Available Programs and Services" or that and "My Applications."
   - [x] Remove displaying # applications within this section
- [ ] There are three possible tags that applications will have in that section:
    - [x] 1. Not yet submitted" (yellow)- shows when an applicant starts, but does not submit an application (maintain our current logic)
    - [x] 2. "Submitted on [date]" (blue) - shows when an application is submitted, but no admin status has been assigned (this might be the only status shown on some applications as not all programs teams use the status feature)
    - [ ] #9165
    - [x] We will not show eligibility badges on program cards in the "My Applications" section
   - [x] Styles and colors changed, see designs
- [ ] Eligibility survey (common intake) also goes into "my applications" when completed
   - [ ] They will be differentiated by the color (blue) of the program card.
   
 ## Design specs
### Spacing guidelines

- Refer to Figma devmode annotations to see the spacing guidelines
- For more information about our spacing guidelines, see our writeup on spacing
    - [CiviForm spacing documentation](https://www.figma.com/design/RmhVJCLgAKWGENJLWWAtNS/CiviForm%3A-Components%2C-Styles?node-id=1614-7047&t=xU4eRWoQtP78C3tV-1)

### Layout guidelines
- Designs provided include mobile, desktop
  - [Desktop layout](https://www.figma.com/design/RmhVJCLgAKWGENJLWWAtNS/CiviForm%3A-Components%2C-Styles?node-id=4702-12978&t=WJyu4u4p7ZNJuSaa-1)
  - [Mobile layout ](https://www.figma.com/design/RmhVJCLgAKWGENJLWWAtNS/CiviForm%3A-Components%2C-Styles?node-id=4702-12978&t=WJyu4u4p7ZNJuSaa-1)

### Color guidelines
- [USWDS theme color tokens](https://designsystem.digital.gov/design-tokens/color/theme-tokens/)
    - All colors used in the application should use one of the USWDS theme tokens
    - Dark text should use always use the theme token `ink`
    - View designs for more details

### Text styles
- All text should use the class `usa-prose` unless specified otherwise
- https://designsystem.digital.gov/components/prose/

### Components used
- [USWDS tag component](https://designsystem.digital.gov/components/tag/)
    - How we use this component
      - We use both the default, `.usa-tag` variant
      - yellow color used is `warning-light`
      - blue color used it `info-light` 
    - We use sentence case spelling on the tags rather than all caps
- [USWDS card component](https://designsystem.digital.gov/components/card/)
  - How we use this component
     - The my applications cards use the default variant with no image
     - We added the USWDS tag component 
     - Common intake submitted cards will use a background color of `primary-lighter`
# New UX
Image

# Old UX
My Applications section

### PR DESCRIPTION
**Pull request by avaleske**
### Description

Set submitted common intake card background color to light blue in northstar.

- Also update tests and add a method to check the submitted tag to ApplicantProgramList.
- This does not intend to style the un-started common intake card.

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [x] Added the correct label: < feature | enhancement | bug | under-development | dependencies | infrastructure | ignore-for-release | database >
- [x] Assigned to a specific person, `civiform/developers`, or a [more specific round-robin list](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#adding-reviewers)
- [x] Added an additional reviewer from outside your organization as FYI (if the primary reviewer is in the same organization as you)
- [x] Removed the release notes section if the title is sufficient for the release notes description, or put more details in that section.
- [x] Created unit and/or browser tests which fail without the change (if possible)
- [x] Performed manual testing (Chrome and Firefox if it includes front-end changes)
- [x] Extended the README / documentation, if necessary. For user-facing features, consider updating [the user docs](https://github.com/civiform/docs). For "under-the-hood" changes or things more relevant to developers, consider updating [the dev wiki](https://github.com/civiform/civiform/wiki).
- [x] Ensured PII wasn't added to any new logs, unless it was guarded by `isDevOrStaging`

#### User visible changes

- [x] Followed steps to [internationalize](https://github.com/civiform/civiform/wiki/Internationalization-%28i18n%29#application-strings) any new strings
  - [x] Added context strings to new [messages](https://github.com/civiform/civiform/blob/main/server/conf/i18n/messages)
  - [x] Didn't use a message in applicant facing code that isn't translated yet (unless behind a flag)
- [x] Wrote browser tests using the [validateAccessibility](https://sourcegraph.com/github.com/civiform/civiform/-/blob/browser-test/src/support/index.ts?L437:14&subtree=true) method
- [x] Made equivalent changes in Thymeleaf for applicant-facing features and changes (or created an issue in the [Northstar epic](https://github.com/orgs/civiform/projects/1/views/94) to track that it's missing from Thymeleaf)
- [x] Tested on mobile view. See [mobile device mode](https://developer.chrome.com/docs/devtools/device-mode/)
- [x] Manually tested at 200% size
- [x] Manually evaluated tab order
- [x] Manually tested with a screen reader if the feature is applicant-facing. See [screen reader testing](https://github.com/civiform/civiform/wiki/Testing#screen-reader-testing)

### Instructions for manual testing

Submit the common intake program and observe that the background color of the submitted tile is blue.

### Issue(s) this completes

Fixes #8724

### CODE REVIEW

**Comment by avaleske** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/civiform/civiform/9311?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#9311** <a href="https://app.graphite.dev/github/pr/civiform/civiform/9311?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/civiform/civiform/9311?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* **#9279** <a href="https://app.graphite.dev/github/pr/civiform/civiform/9279?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#9297** <a href="https://app.graphite.dev/github/pr/civiform/civiform/9297?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: server/app/views/applicant/ProgramCardsSectionFragment.html [hunk 1]
```diff
@@ -34,7 +34,10 @@
   class="usa-card tablet-lg:grid-col-6 widescreen:grid-col-4 cf-application-card"
   th:classappend="${showImage} ? usa-card--header-first : '' "
 >
-  <div class="usa-card__container">
+  <div
+    class="usa-card__container"
+    th:classappend="${card.isCommonIntakeForm} ? bg-primary-lighter : '' "
```
Comment (melanie-exygy) [inline server/app/views/applicant/ProgramCardsSectionFragment.html:39]: **suggestion (non-blocking):** We might want to also add the condition that the card `sectionType` is 'MY_APPLICATIONS'.  Then, it'll be done for Anna when she works on the un-submitted intake form.

File: browser-test/src/applicant/navigation/northstar_application_navigation_common_intake.test.ts [hunk 2]
```diff
@@ -220,12 +223,19 @@ test.describe('Applicant navigation flow', {tag: ['@northstar']}, () => {
...
+            CardSectionName.MyApplications,
+            commonIntakeProgramName,
+          )
+          .locator('div.bg-primary-lighter'),
+      ).toBeVisible()
+      await applicantProgramList.expectSubmittedTag(
+        CardSectionName.MyApplications,
+        commonIntakeProgramName,
       )
+      await validateAccessibility(page)
```
Comment (melanie-exygy) [inline browser-test/src/applicant/navigation/northstar_application_navigation_common_intake.test.ts:238]: **praise:** Thanks for remembering this!

File: server/app/views/applicant/ProgramCardsSectionFragment.html (hunk 1 above)
Comment (avaleske) [reply server/app/views/applicant/ProgramCardsSectionFragment.html:39]: i think the unsubmitted card is the same color, actually. It just spans the full width of the page.

**Pull request merged**